index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Static Web Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+  <article>
+    <header>
+      <h1>Thomas</h1>
+    </header>
+    <section>
+      <div class = "image">
+        <img src="cats-1.jpg" alt="Cat-1" height="350" width="420">
+      </div>
+      <div class="description">
+        Climb leg stare at ceiling light, so lick sellotape or destroy couch, yet hide at bottom of staircase to trip human. This human feeds me, i should be a god stretch. Mew chase after silly colored fish toys around the house or mark territory, for chase the pig around the house. Jump launch to pounce upon little yarn mouse, bare fangs at toy run hide in litter box until treats are fed stretch stretch. Climb a tree, wait for a fireman jump to fireman then scratch his face meow all night having their mate disturbing sleeping humans, touch water with paw then recoil in horror and cat is love, cat is life and swat turds around the house so lay on arms while you're using the keyboard. Stares at human while pushing stuff off a table knock over christmas tree stretch. Thinking longingly about tuna brine poop in litter box, scratch the walls flee in terror at cucumber discovered on floor kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff cough furball so all of a sudden cat goes crazy. Damn that dog . Peer out window, chatter at birds, lure them to mouth chase the pig around the house. Eat a plant, kill a hand destroy the blinds, but eat owner's food yet soft kitty warm kitty little ball of furr but lay on arms while you're using the keyboard. Play time lick arm hair. Lick butt see owner, run in terror and you call this cat food?. Soft kitty warm kitty little ball of furr attack feet, or flop over, so sit in box jump off balcony, onto stranger's head use lap as chair. Hunt by meowing loudly at 5am next to human slave food dispenser hunt anything that moves.
+      </div>
+      <div class="availability">
+        Available
+      </div>
+    </section>
+    <section>
+      <header>
+        <h3>Specifications</h3>
+      </header>
+      <div class="specifications">
+        Size:  Medium
+      </div>
+      <div class="specifications">
+        Weight:  4.5 lbs.
+      </div>
+    </section>
+      <footer>
+        Specifications may vary depending on quantity and types of cats ordered.
+      </footer>
+    <section>
+      <header class="specifications"><h4>Price:</h4></header>
```

Comment:
I'm assuming this is the first card you made, and that the others are all based on the second card.  Take a look at [line 66](https://github.com/nss-evening-cohort-04/mastery-exercise-html-jkillz2020/pull/1/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR66).  Spot the difference?  Do you see why doesn't this way work?